inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -397,27 +397,27 @@ CREATE TABLE IF NOT EXISTS `source_file_detail`
 (
-    `id`                 int(11)      NOT NULL AUTO_INCREMENT COMMENT 'ID',
-    `inlong_group_id`    varchar(256) NOT NULL COMMENT 'Inlong group id',
-    `inlong_stream_id`   varchar(256) NOT NULL COMMENT 'Inlong stream id',
-    `source_name`        varchar(128) NOT NULL DEFAULT '' COMMENT 'source_name',
-    `source_type`        varchar(20)           DEFAULT '0' COMMENT 'Source type, including: FILE, DB, etc',
-    `agent_ip`           varchar(40)           DEFAULT NULL COMMENT 'Ip of the agent running the task',
-    `uuid`               varchar(30)           DEFAULT NULL COMMENT 'Mac uuid of the agent running the task',
-    `data_node_name`     varchar(128)          DEFAULT NULL COMMENT 'Node name, which links to data_node table',
-    `cluster_id`         int(11)               DEFAULT NULL COMMENT 'Id of the cluster that collected this source',
-    `serialization_type` varchar(20)           DEFAULT NULL COMMENT 'Serialization type, support: csv, json, canal, avro, etc',
-    `snapshot`           text                  DEFAULT NULL COMMENT 'Snapshot of this source task',
-    `report_time`        timestamp    NULL COMMENT 'Snapshot time',
-    `ext_params`         text                  DEFAULT NULL COMMENT 'Another fields will be saved as JSON string, such as filePath, dbName, tableName, etc',
-    `version`            int(11)               DEFAULT '1' COMMENT 'Stream source version',
-    `status`             int(4)                DEFAULT '110' COMMENT 'Data source status',
-    `previous_status`    int(4)                DEFAULT '110' COMMENT 'Previous status',
-    `is_deleted`         int(11)               DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, > 0: deleted',
-    `creator`            varchar(64)  NOT NULL COMMENT 'Creator name',
-    `modifier`           varchar(64)           DEFAULT NULL COMMENT 'Modifier name',
-    `create_time`        timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Create time',
-    `modify_time`        timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Modify time',
+    `id`                  int(11)      NOT NULL AUTO_INCREMENT COMMENT 'ID',
+    `inlong_group_id`     varchar(256) NOT NULL COMMENT 'Inlong group id',
+    `inlong_stream_id`    varchar(256) NOT NULL COMMENT 'Inlong stream id',
+    `source_name`         varchar(128) NOT NULL DEFAULT '' COMMENT 'source_name',
+    `source_type`         varchar(20)           DEFAULT '0' COMMENT 'Source type, including: FILE, DB, etc',
+    `agent_ip`            varchar(40)           DEFAULT NULL COMMENT 'Ip of the agent running the task',
+    `uuid`                varchar(30)           DEFAULT NULL COMMENT 'Mac uuid of the agent running the task',
+    `data_node_name`      varchar(128)          DEFAULT NULL COMMENT 'Node name, which links to data_node table',
+    `inlong_cluster_name` varchar(128)          DEFAULT NULL COMMENT 'Cluster name of the agent running the task',
```

Comment:
Please change the param in the `manager-test` module at the same time.

Reply:
ok